libs/core/src/lib/select/select.component.html

```diff
@@ -41,6 +40,9 @@
         [attr.id]="controlId"
         [attr.aria-active]="_isOpen"
         aria-live="polite"
+        role="combobox"
+        aria-autocomplete="list"
```

Comment:
why do you have `aria-autocomplete`? 
The aria-autocomplete attribute indicates whether inputting text could trigger display of one or more predictions of the user's intended value for a combobox, searchbox or  textbox and specifies how predictions will be presented if they are made. In Select we don't type. 
For more information: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-autocomplete

Reply:
@InnaAtanasova yeah, my bad. Will remove this one